From acd3e6e7c797861b7e9cb639b4b52bc9a6a2309e Mon Sep 17 00:00:00 2001
From: Pete O_o <pmb@meta.com>
Date: Fri, 25 Jul 2025 10:18:00 -0700
Subject: [PATCH] 82029: meta-facebook: catalina: Remove exit on second FPGA
 ready check failure | https://gerrit.openbmc.org/c/openbmc/openbmc/+/82029
 Summary: 82029: meta-facebook: catalina: Remove exit on second FPGA ready
 check failure | https://gerrit.openbmc.org/c/openbmc/openbmc/+/82029

Downstreaming this until yocto sync is working again

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:
---
 standby-power-enable | 1 -
 1 file changed, 1 deletion(-)

diff --git a/standby-power-enable b/standby-power-enable
index 6625d65..0ed5270 100644
--- a/standby-power-enable
+++ b/standby-power-enable
@@ -185,7 +185,6 @@ fi
 
 if ! wait_gpio_value "SEC_FPGA_READY_BMC" "$GPIO_HIGH" 180 1; then
     echo "Error: failed to get SEC_FPGA_READY_BMC as high (1) in 180 sec"
-    exit 1
 fi
 
 bind_fru_eeproms
